package.json

```diff
@@ -82,22 +82,34 @@
       "category": "Utility"
     }
   },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx}": [
+      "npm run lint:fix",
+      "prettier --write"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
```

Comment:
I might be wrong, but is this redundant since we have `.husky/pre-commit` file?

Reply:
Right you are. https://github.com/elastic/synthetics-recorder/pull/177/commits/b65e0afebb26740796178727f9b7349e3705c403